app/client/sagas/creditTransferSagas.ts

```diff
@@ -62,7 +62,8 @@ function* updateStateFromCreditTransfer(result: CreditLoadApiResult) {
     // we need to add the newly created credit_transfer id
     // to the associated transfer_account object credit_transfer array
     yield put({
-      type: UPDATE_TRANSFER_ACCOUNTS_CREDIT_TRANSFERS,
+      type:
+        TransferAccountActionTypes.UPDATE_TRANSFER_ACCOUNTS_CREDIT_TRANSFERS,
       credit_transfer_list
     });
```

Comment:
Same here, should use an action `TransferAccountAction.updateTransferAccountsCreditTransfers`